web/portal/client/src/entities/Ddi/Ddi.tsx

```diff
@@ -187,6 +193,10 @@ const properties: DdiProperties = {
   retailAccount: {
     label: _('Retail Account', { count: 1 }),
   },
+  locution: {
+    label: _('Voiceover and hang up', { count: 1 }),
+    null: _('Direct hangup'),
```

Comment:
The field must be required, like in the other route types of DDI, so no null option must be available.